Scripts/Generator/MapLoader.cs

```diff
@@ -59,11 +73,12 @@ public ShaderMaterial FloorMaterial
         set
         {
             floorMaterial = value;
-            if (ready)
-                RebuildMap();
+            RebuildMap();
         }
     }
 
+    private bool preview;

```

Comment:
Would it be better to call this showPreview and ShowPreview since its a bool?

Reply:
Yea I couldn't think of a good name